public/scripts/power-user.js

```diff
@@ -4072,4 +4072,34 @@ $(document).ready(() => {
         ],
         helpString: 'activates a movingUI preset by name',
     }));
+    SlashCommandParser.addCommandObject(SlashCommand.fromProps({
+        name: 'stop-strings',
+        aliases: ['stopping-strings'],
+        helpString: 'Sets a list of custom stopping strings. Gets the list if no value is provided.',
```

Comment:
I'd add an info in the help text that if `STRICT_ESCAPING` is enabled - which most people should have -, you need to make sure to escape all pipes, even inside the JSON.